website/docs/querying.mdx

```diff
@@ -33,15 +33,23 @@ let subscription = Selection.Subscription {
     return list
 }
 
-// Start the event.
-let task = listen(for: subscription, on: "ws://localhost:4000/graphql") { result in
+// Initialize a socket
+let socket = GraphQLSocket<URLSessionWebSocketTask>(.init(
+    url: URL(string: "ws://localhost:4000/graphql")!,
+    headers: [:],
+    autoConnect: true
+))
+var cancellable: SocketCancellable?
+
+// Open the socket and start listening
+cancellable = socket.subscribe(to: subscription) { result in
     if let data = try? result.get() {
         print(data)
     }
 }
 
-// To close the socket.
-task.close()
+// Close the socket.
+cancellable = nil
```

Comment:
or cancellable?.cancel() which is more readable imo

Reply:
AFAIK it cancels it's publisher before deinitialization, the point is to close the socket as well

Reply:
I doubled it from the StarWars example, let me dive deeper into how it works...

Reply:
My guess that there is also missing explicit `socket?.stop()` after that